packages/DataTable/src/hooks/useIsUpdated.js

```diff
@@ -0,0 +1,7 @@
+import usePrevious from "@paprika/helpers/lib/hooks/usePrevious";
+
+export default function useIsUpdated(prop) {
```

Comment:
`nip`

This is no exactly a custom hook, there is no state to manage here,  but this could totally be a helper function

export default function(prop, prevProp) {
  return prevProp !== undefined && prop !== prevProp;
}

but of course you will have to import usePrevious in each file.

Reply:
But there's state management in `usePrevious`
Isn't it just like this? https://github.com/streamich/react-use/blob/master/src/usePreviousDistinct.ts